packages/rich-text/src/component/index.native.js

```diff
@@ -140,11 +145,15 @@ export class RichText extends Component {
 	 */
 	createRecord() {
 		const { preserveWhiteSpace } = this.props;
+		const htmlForCreate = this.getHtmlForRecordCreation(
```

Comment:
Maybe only do this for Android?

Reply:
limited to Android here ca2cf3b70a8261a2907c68552be84f160f64d9a1